it/mail/chrome/messenger/addressbook/addressBook.properties

```diff
@@ -103,75 +103,75 @@ stateZipSeparator=
 prefixTo=A
 prefixCc=CC
 prefixBcc=Ccn
-emptyEmailCard=Please remove the following cards with no email address : 
+emptyEmailCard=Si prega di rimuovere i seguenti contatti che non presentano indirizzi e-mail:
 addressBook=Rubrica
 
 # mailnews.js
 ldap_2.servers.pab.description=Rubrica personale
-ldap_2.servers.history.description=Indirizzi collezionati
+ldap_2.servers.history.description=Indirizzi raccolti
 ## LOCALIZATION NOTE (ldap_2.servers.osx.description is only used on Mac OS X)
 ldap_2.servers.osx.description=Rubrica di Mac OS X
 
 # status bar stuff
-## LOCALIZATION NOTE (totalCardStatus): 
+## LOCALIZATION NOTE (totalCardStatus):
 ## %1$S is address book name, %2$S is card count
-totalCardStatus=Total Cards in %1$S: %2$S
+totalCardStatus=Contatti totali in %1$S: %2$S
 ## LOCALIZATION NOTE (matchesFound):  do not localize %S
 noMatchFound=Nessuna corrispondenza
-matchFound=una corrispondenza trovata
+matchFound=1 corrispondenza trovata
 matchesFound=%S corrispondenze trovate
 
-cardsCopied=%S cards copied
-cardCopied=1 card copied
-cardsMoved=%S cards moved
-cardMoved=1 card moved
+cardsCopied=%S contatti copiati
+cardCopied=1 contatto copiato
+cardsMoved=%S contatti spostati
+cardMoved=1 contatto spostato
 
 # LDAP directory stuff
 directoriesListItemNone=Nessuno
-invalidName=Inserire un nome valido.
-invalidHostname=Inserire un nome di host valido.
-invalidPortNumber=Inserire un numero di porta valido.
-invalidResults=Inserire un numero valido nel campo risultati.
-abReplicationOfflineWarning=È indispensabile essere collegato per eseguire una replica LDAP.
-abReplicationSaveSettings=È indispensabile salvare le impostazioni prima che una cartella possa essere scaricata.
+invalidName=Si prega di inserire un nome valido.
+invalidHostname=Si prega di inserire un nome di host valido.
+invalidPortNumber=Si prega di inserire un numero di porta valido.
+invalidResults=Si prega di inserire un numero valido nel campo risultati.
+abReplicationOfflineWarning=Devi essere online per eseguire una replica LDAP.
+abReplicationSaveSettings=È necessario salvare le impostazioni prima di poter scaricare una cartella.
 
 # For importing / exporting
-ExportAddressBookTitle=Esporta Rubrica
+ExportAddressBookTitle=Esporta rubrica
 LDIFFiles=LDIF
 CSVFiles=Separati da virgola
 TABFiles=Separati da tabulazione
 failedToExportTitle=Esportazione non riuscita
 failedToExportMessageNoDeviceSpace=Esportazione rubrica non riuscita, spazio esaurito nel dispositivo.
-failedToExportMessageFileAccessDenied=Esportazione rubrica non riuscita, accesso al file negato.
+failedToExportMessageFileAccessDenied=Esportazione rubrica non riuscita, accesso negato al file.
 
 # For getting authDN for replication using dlg box
-AuthDlgTitle=Replicazione rubrica LDAP
-AuthDlgDesc=Per aver accesso alla rubrica remota, inserire nome utente e password.
+AuthDlgTitle=Replica rubrica LDAP
+AuthDlgDesc=Per accedere al server di directory, inserire nome utente e password.
```

Comment:
CORRECTIONS NEEDED. I would shift the main clause before the subordinate, as follows: 

 +AuthDlgDesc=Inserire nome utente e password per accedere al server di directory.